docs/README.md

```diff
@@ -8,6 +8,11 @@ This directory contains all of the documentation relating to NGINX Kubernetes Ga
 - [Gateway API Compatibility](gateway-api-compatibility.md): Describes which Gateway API resources NGINX Kubernetes
 Gateway supports and the extent of that support.
 - [Installation](installation.md): Walkthrough on how to install NGINX Kubernetes Gateway on a generic Kubernetes cluster.
+- Guides:
```

Comment:
We have a `directories` section below. Either this should be there or we reorganize a bit.

Reply:
I went with that approach first. However, my concern is those guides are bit hidden this way :(  is there a good way to address it?

happy to make it consistent though, and add it as a directory

Reply:
I guess we could pull the Directories up into this section and just add in the description that they are directories.

Reply:
after more thinking, I think the guides are not that hidden on the docs index page